### PR TITLE
Update rn for Native Template 10

### PR DESCRIPTION
**Pull request by UrazAkgultan**

### CODE REVIEW

**Comment by ConnorLand** [comment]
Hi @UrazAkgultan , why is this marked as draft? Do you plan to make more commits?

**Comment by ConnorLand** [comment]
> Hi @UrazAkgultan , why is this marked as draft? Do you plan to make more commits?

On Slack he said it's fine to merge.

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/_index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "Studio Pro 9 and 10 compatible Native Template release notes."
 
 These are the current Native Template versions for Studio Pro 9 in active development:
 
-* Native Template versions of [7](/releasenotes/mobile/nt-7-rn/) are compatible with apps built using the following versions of Studio Pro:
```
Comment (stelselim) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/_index.md:10]: Could it be a bit confusing for Studio Pro 9.24? We only support **the native template version 8** for **Studio Pro 9.24**. Versions 8 - 9 - 10 compatibility starts from Studio Pro 10. Ex: Native Template 10 is compatible with Studio Pro 10.17.

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/_index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: "Studio Pro 9 and 10 compatible Native Template release notes."
...
+* Native Template versions of [7](/releasenotes/mobile/nt-7-rn/), [8]((/releasenotes/mobile/nt-8-rn/)), [9]((/releasenotes/mobile/nt-9-rn/)), and [10](/releasenotes/mobile/nt-10-rn/) are compatible with apps built using the following versions of Studio Pro:
```
Comment (stelselim) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/_index.md]: Could it be a bit confusing for Studio Pro 9.24? We only support the native template version 8 for Studio Pro 9.24. Versions 8 - 9 - 10 compatibility starts from Studio Pro 10. Ex: Native Template 10 is compatible with Studio Pro 10.17.

Comment (ConnorLand) [reply content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/_index.md]: Good point @stelselim . Jonathan Diehl and I will address this confusion in a subsequent PR clarifying these parent docs.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**